tailscale/install/mint-leaf.yml

```diff
@@ -0,0 +1,28 @@
+name: tailscale/install
+version: 1.0.0
+description: Install Tailscale
+source_code_url: https://github.com/rwx-research/mint-leaves/tree/main/tailscale/install
+issue_tracker_url: https://github.com/rwx-research/mint-leaves/issues
+
+parameters:
+  version:
+    description: 'Version to install'
+    default: "latest"
+
+tasks:
+  - key: download
+    run: |
+      if [ "$VERSION" = "latest" ]; then
+        VERSION=$(curl -s "https://pkgs.tailscale.com/stable/?mode=json" | jq -r .Version)
+      fi
+      TAILSCALE_DIR="tailscale_${VERSION}_amd64"
+
+      curl -L https://pkgs.tailscale.com/stable/${TAILSCALE_DIR}.tgz -O
```

Comment:
If this curl command fails something else would fail downstream, but the error message will be more clear if we add `-f` to this